From 9575c64a751022d2e64b0e63d055b28ec8aa8bb7 Mon Sep 17 00:00:00 2001
From: Marcin Wojtas <mw@semihalf.com>
Date: Fri, 12 Mar 2021 08:17:35 +0000
Subject: [PATCH] Do not force local BinDir directory

By default the system.conf file forces a relative path
to the ippsample tools directory. In case the ippsample
is properly installed in the system this setting is not necessary
and becomes problematic in standard usage. Update configuration file
and rely on the paths set dynamically during server configuration.
---
 test/system.conf | 1 -
 1 file changed, 1 deletion(-)

diff --git a/test/system.conf b/test/system.conf
index a30f7a8..1469399 100644
--- a/test/system.conf
+++ b/test/system.conf
@@ -1,4 +1,3 @@
-BinDir tools
 DefaultPrinter ipp-everywhere-pdf
 KeepFiles Yes
 LogFile stderr
-- 
2.31.0.rc2.261.g7f71774620-goog

